src/dotnet/APIView/APIView/Model/V2/ReviewLine.cs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using APIView.TreeToken;
+
+namespace APIView.Model.V2
+{
+    /*** Review line object corresponds to each line displayed on API review. If an empty line is required then 
+     * add a review line object without any token. */
+    public class ReviewLine
+    {
+        /*** LineId is only required if we need to support commenting on a line that contains this token. 
+         *  Usually code line for documentation or just punctuation is not required to have lineId. lineId should be a unique value within 
+         *  the review token file to use it assign to review comments as well as navigation Id within the review page.
+         *  for e.g Azure.Core.HttpHeader.Common, azure.template.template_main
+         */
+        public string LineId { get; set; }
+        public string CrossLanguageId { get; set; }
```

Comment:
How is LineId different than CrossLanguageId?

Reply:
LineID is language dependent ID and CrossLanguageId is language independent ID to support cross language view from TypeSpec originated SDKs to map a TypeSpec API definition to SDK API. This is yet to be implemented for .NET.

Reply:
Should CrossLanguageId be named something like `SourceSpecId`?

Reply:
Let me check.  I will check with Travis and cross language project team and make this change.

Reply:
It's also can be used to refer across languages without spec as well. So I think it's probably fine to leave as it is for now.

Reply:
we can always rename when we look at this more closely